runtime/doc/pattern.txt

```diff
@@ -141,7 +141,7 @@ CTRL-C			Interrupt current (search) command.  Use CTRL-Break on
 			executing autocommands |autocmd-searchpat|.
 			Same thing for when invoking a user function.
 
-While typing the search pattern the current match will be shown if the
+While typing the search pattern the current match will be shown as long as the
```

Comment:
It still reads fine as "if". I mention this to avoid however many other similar doc changes may result from defaults changes.